crates/re_build_tools/src/lib.rs

```diff
@@ -85,6 +85,7 @@ pub fn export_env_vars() {
         );
     }
 
+    // TODO(paris): Determine if this needs to be updated if we're displaying in the local timezone.
```

Comment:
This is used for etching build time into the build itself and should stay in UTC.

Reply:
Sounds good, thanks.